Gemfile

```diff
@@ -153,8 +155,9 @@ group :test do
 end
 
 group :development, :test do
-  gem 'pry', '~> 0.9.12.6'
-  gem 'pry-rails', '~> 0.3.2'
-  gem 'pry-nav', '~> 0.2.3'
-  gem 'pry-stack_explorer', '~> 0.4.9.1'
+  gem 'pry-byebug'
+  gem 'awesome_rails_console', '~> 0.4.0'
 end
+
+
+gem 'therubyracer', platforms: :ruby
```

Comment:
No version number here. Is that desired? `bundle update` can now update ruby racer to any version.